charts/newrelic-infrastructure-v3/templates/kubelet/integrations-configmap.yaml

```diff
@@ -7,11 +6,32 @@ metadata:
     {{- include "newrelic.labels" . | nindent 4 }}
   name: {{ template "newrelic.fullname" . }}-integrations-cfg
 data:
+  # This ConfigMap holds config files for integrations. They should have the following format:
+  #redis-config.yml: |
+  #  # Run auto discovery to find pods with label "app=redis"
+  #  discovery:
+  #    command:
+  #      # Run discovery for Kubernetes. Use the following optional arguments:
```

Comment:
why is this useful? Do we point here from docs?

Reply:
isn't better somewhere else? Templates with those info are weird

Reply:
Yes, in de docs we (now) say:

> In order for the snippet above to be recognized by the integration, it must be placed inside the designated ConfigMap. If you are using our Kubernetes Integration v3, a ConfigMap should already be generated, with a name ending in `-integrations-cfg`. Locate the config map and add the modified snippet to it, so it ends up looking like this:

I thought that having a commented-out example in there would make extra clear to the user that's the ConfigMap they need to edit.

Reply:
mmm 👍  I understand we do not follow normal flow since we point users not using helm to check on helm charts template.

I am fine with it